demos/image_processing_demo/cpp/description/README.md

```diff
@@ -0,0 +1,94 @@
+# Image Processing C++ Demo
+
+This demo processes the image according to the selected type of processing. At this moment demo can work with the next types:
+
+* `super_resolution`
+* `deblurring`
+
+## Examples
+
+Exmaple for deblurring type (left - source image, right - image after deblurring):
+
+![](./deblurred_image.png)
+
+Example for super_resolution type:
+
+Low resolution:
+
+![](./street_640x360.png)
+
+Bicubic interpolation:
+
+![](./street_resized.png)
+
+Super resolution:
+
+![](./street_resolution.png)
+
+## How It Works
+
+Before running the demo, user must choose type of processing and model for this processing. \
+For `super_resolution` user can choose the next models:
+
+* [single-image-super-resolution-1032](../../../../models/intel/single-image-super-resolution-1032/description/single-image-super-resolution-1032.md) -  It enhances the resolution of the input image by a factor of 4.
+* [single-image-super-resolution-1033](../../../../models/intel/single-image-super-resolution-1033/description/single-image-super-resolution-1033.md) -  It enhances the resolution of the input image by a factor of 3.
+* [text-image-super-resolution-0001](../../../../models/intel/text-image-super-resolution-0001/description/text-image-super-resolution-0001.md) -  A tiny model to 3x upscale scanned images with text.
+
+For `deblurring` user can use [deblurgan-v2](../../../../models/public/deblurgan-v2/deblurgan-v2.md) - generative adversarial network for single image motion deblurring.
+
+The demo runs inference and shows results for each image captured from an input. Depending on number of inference requests processing simultaneously (-nireq parameter) the pipeline might minimize the time required to process each single image (for nireq 1) or maximizes utilization of the device and overall processing performance.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+
+```
+./image_processing_demo -h
+[ INFO ] InferenceEngine: <version>
+
+image_processing_demo_async [OPTION]
+Options:
+
+    -h                        Print a usage message.
+    -pt "<type>"              Required. Type of processing: super_resolution, deblurring
```

Comment:
all other demos use -at to select model type, it is better to be consistent with demo command line api